registry/lib/components/style/dark-mode/dark-slice-18.scss

```
@@ -571,3 +571,22 @@
     }
   }
 }
+.home-cell-desc-title {
+  @include color("ffffff");
+}
+.home-cell-desc-subtitle {
+  @include color("ffffff80");
+}
```

关于文字颜色的修改, 原先是黑色的应改为 `#eee`, 原先是灰色的应改为 `#aaa`
```suggestion
.home-cell-desc-title {
  @include color("e");
}
.home-cell-desc-subtitle {
  @include color("a");
}
```